orga/tests/unit/services/url_test.js

```diff
@@ -164,7 +164,7 @@ module('Unit | Service | url', function (hooks) {
     test('should get "pix.org" french url when current language is fr', function (assert) {
```

Comment:
Remarque : on pourrait remplacer should get par 'returns' non ?

Reply:
C'est fait. J'en ai profité pour l'appliquer à tous les tests du fichier 😄

Reply:
Cool !